Description: remove unnecessary dev dependencies
 The dependency on webpki-roots is only used to compare system certificates
 with the ones from webpki-roots, which doesn't make much sense in an
 automated test during packaging.
Author: Jan Niehusmann <jan@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Last-Update: 2021-05-12

--- rust-rustls-native-certs-0.5.0.orig/Cargo.toml
+++ rust-rustls-native-certs-0.5.0/Cargo.toml
@@ -33,9 +33,6 @@ version = "0.7.0"
 [dev-dependencies.webpki]
 version = "0.21"
 
-[dev-dependencies.webpki-roots]
-version = "0"
-
 [features]
 default = ["rustls"]
 [target."cfg(all(unix, not(target_os = \"macos\")))".dependencies.openssl-probe]
